libs/designsystem/calendar/src/helpers/calendar-cell.model.ts

```diff
@@ -1,9 +1,18 @@
-export interface CalendarCell {
+export interface CalendarCell extends CalendarCellMetadata {
```

Comment:
Consider renaming - it feels like this interface represents a day now more than a cell, also given that `cssClasses` has been removed:
```suggestion
export interface CalendarDay extends CalendarDayMetadata {
```